src/model/operator.ts

```diff
@@ -5,22 +5,27 @@ export type Operator = {
   id: string
 }
 
-async function getOperatorId(name: string) {
+export const RELEVANT_OPERATORS = async () => {
```

Comment:
you can use IIFE to make `RELEVANT_OPERATORS` a promise. it could be nice, because it's reusable this way.

Reply:
Ta first place it was 8 calls. Like screenshot at issue

Reply:
I see that it had many calls, I can't see why